wgsl/index.bs

```diff
@@ -4439,7 +4439,7 @@ TODO: deduplicate these tables
   <tr><td>*e* : *T*, *T* is *FloatVec*<td>`isFinite(e)` : bool<*N*>, where *N = Arity(T)*<td>OpIsFinite, or emulate
   <tr><td>*e* : f32<td>`isNormal(e)` : bool<td>OpIsNormal
   <tr><td>*e* : *T*, *T* is *FloatVec*<td>`isNormal(e)` : bool<*N*>, where *N = Arity(T)*<td>OpIsNormal, or emulate
-  <tr><td>*e* : array<*E*><td>`arrayLength(e)` : u32<td>OpArrayLength
+  <tr><td>*e* : ptr&lt;storage,array&lt;*E*&gt;&gt;<td>`arrayLength(e)` : u32<td>OpArrayLength
```

Comment:
This seems confusing, if I want to say:

```rust
struct S {
  a : array<i32>;
}
[[group(0), set(0)]] var<storage> c : S;

fn vtx() -> void {
  var b : u32 = arrayLength(c.a);
}
```

would seem from the reading like it isn't allowed, but just that the var is an implicit pointer?